src/tag.ts

```diff
@@ -4315,4 +4315,9 @@ export const Tag: { [key: string]: number } = {
     Item: 0xFFFEE000,
     ItemDelimitationItem: 0xFFFEE00D,
     SequenceDelimitationItem: 0xFFFEE0DD,
+
+    // private tags
+     PhilipsSUVScaleFactor: 0x70531000,
```

Comment:
I would recommend adding the Private Creator Data Elements, 0x70530010, here.

Reply:
@richterjens please add according to Fredrik's request. Approving otherwise.